layers/+lang/lua/README.org

```diff
@@ -36,24 +36,34 @@ In order to enable code linting, install [[https://github.com/mpeterv/luacheck][
 
 ** Backends
 Supported backends are:
-- =lsp-emmy= using clangd LSP server
+- =lsp-emmy-lua= using emmy LSP server
+- =lsp-lua-language-server= using lua-language-server.
+- =lsp-lua-lsp= using lua-lsp.
 - =lua-mode= using emacs package
 
 *** LSP
-LSP support is provided via the [[file:../../+tools/lsp/README.org][LSP layer]], currently only [[https://github.com/EmmyLua/EmmyLua-LanguageServer][EmmyLua]] is supported.
+LSP support is provided via the [[file:../../+tools/lsp/README.org][LSP layer]], currently [[https://github.com/EmmyLua/EmmyLua-LanguageServer][EmmyLua]], [[https://github.com/sumneko/lua-language-server][lua-language-server]] and [[https://github.com/Alloyed/lua-lsp][lua-lsp]] are supported.
 
-To use the =lsp-emmy= backend, please download the [[https://github.com/EmmyLua/EmmyLua-LanguageServer][EmmyLua-LS-all.jar]] and put it into =~/.emacs.d/=,
-then set the layer variable lua-backend to =lsp-emmy=.
+To use the =lsp-emmy-lua= backend, please download the [[https://github.com/EmmyLua/EmmyLua-LanguageServer][EmmyLua-LS-all.jar]] and put it into =~/.emacs.d/=,
+then set the layer variable lua-backend to =lsp-emmy-lua=.
```

Comment:
then -> and then

This is a grammar mistake. See https://www.wikiwand.com/en/Sentence_clause_structure#/Run-on_sentences

Reply:
resolved in new submition.